TCL4 Data Management/utm-tcl4-data-collection.yaml

```diff
@@ -657,13 +630,15 @@ definitions:
       - "exchanged_data_type"
       - "source_uss"
       - "target_uss"
-      - "uss_name"
+      - "metaDataDmpUss"
```

Comment:
move this to the top of the required elements list.

Reply:
do this anywhere it is a required element... metadata first makes sense I think.

Reply:
got it will do

@lmadhavrao its been blessed - let's lock it in and go

Reply:
changed implemented